components/atom/switch/demo/index.js

```diff
@@ -111,23 +111,33 @@ const TypesArticle = () => (
       This package gives 3 different <Code>type</Code> values provided under the{' '}
```

Comment:
We should update this line as well

Reply:
there are only 3 different types, but 'single' one have different behaviors

Reply:
We have improved the demo, to show clearly the 3 types, and the two behaviors of the SINGLE type.

<img width="538" alt="Captura de pantalla 2021-08-25 a las 8 37 43" src="https://user-images.githubusercontent.com/17734680/130739038-307cfba1-2eb3-4dc9-83e4-4bb182a0e393.png">

Reply:
great!